extension/lib/main.js

```diff
@@ -113,14 +110,65 @@ exports.main = function (options, callbacks) {
         widget.tooltip = config.extension.widget_tooltips[data];
         break;
     }
+  });*/
+
+  var { Frame } = require("sdk/ui/frame");
+  var { ToggleButton } = require('sdk/ui/button/toggle');
+  var { Toolbar } = require("sdk/ui/toolbar");
+
+  var frame = new Frame({
+    url: "./frame/frame.html"
   });
 
+  var menuButton = ToggleButton({
+    id: "menu-button",
+    label: "Menu",
+    icon: {
+      "32" : "./addon32.png",
+      "64" : "./addon64.png"
+    },
+    onChange: handleMenuButtonChange
+  });
+
+  var logButton = ToggleButton({
+    id: "log-button",
+    label: "Logging",
+    icon: {
+      "32" : "./addon32.png",
+      "64" : "./addon64.png"
```

Comment:
These images have transparent background.

This is `logging_disabled.png`:
![logging_disabled](https://cloud.githubusercontent.com/assets/568751/8241477/d8fd1176-1609-11e5-8bea-c1fa51242592.png)

This is `logging_enabled.png`:
![logging_enabled](https://cloud.githubusercontent.com/assets/568751/8241476/d8fb517e-1609-11e5-82a0-2dbaec765fcd.png)